apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,367 @@
+                            break;
+                        default:
+                            throw new ArgumentOutOfRangeException();
+                    }
+                }
+            }
+            // End snippet
+
+            await BigtableAssert.HasNoValuesAsync(
+                client,
+                new TableName(projectId, instanceId, tableId),
+                "user23456");
+        }
+
+        [Fact]
+        public async Task ReadModifyWriteRowAsync()
+        {
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                Mutations.SetCell("Score", "Level 1", 12));
+
+            // Snippet: ReadModifyWriteRowAsync(TableName,BigtableByteString,*)
+            // Increase the value of the cell with the "Level 1" qualifier in the
+            // "Score" column family by 55.
+            ReadModifyWriteRowResponse response = await client.ReadModifyWriteRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                ReadModifyWriteRules.Increment("Score", "Level 1", 55));
+
+            Cell modifiedCell = response.Row.Families[0].Columns[0].Cells[0];
+
+            // Implicitly convert the ByteString Value into a BigtableByteString,
+            // which supports additional conversions specific to Bigtable.
+            BigtableByteString value = modifiedCell.Value;
+
+            // Convert the BigtableByteString to a long to obtain the updated value.
+            long newScore = (long)value;
+
+            // End snippet
+
+            Assert.Equal(67, newScore);
+        }
+
+        [Fact]
+        public async Task ReadRowAsync()
+        {
+            var cts = new CancellationTokenSource();
+            var cancellationToken = cts.Token;
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                Mutations.DeleteFromRow(),
+                Mutations.SetCell("Address", "Country", "USA"),
+                Mutations.SetCell("Address", "Line 1", "1234 Main Street"),
+                Mutations.SetCell("Address", "Line 2", "Apt. 2"),
+                Mutations.SetCell("Score", "Level 1", 12));
+
+            // Snippet: ReadRowAsync(TableName,BigtableByteString,RowFilter,CallSettings)
+            // Only read cells from the "Line 1" and "Line 2" columns in the
```

Comment:
Just to clarify, it's really "anything starting with line and a digit" but we know that there *are* only Line 1 and Line 2, yes?

Reply:
Good point. Updated.